aws-rds-optiongroup/src/main/java/software/amazon/rds/optiongroup/Translator.java

```diff
@@ -0,0 +1,220 @@
+package software.amazon.rds.optiongroup;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import software.amazon.awssdk.services.rds.model.AddTagsToResourceRequest;
+import software.amazon.awssdk.services.rds.model.CreateOptionGroupRequest;
+import software.amazon.awssdk.services.rds.model.DBSecurityGroupMembership;
+import software.amazon.awssdk.services.rds.model.DeleteOptionGroupRequest;
+import software.amazon.awssdk.services.rds.model.DescribeOptionGroupsRequest;
+import software.amazon.awssdk.services.rds.model.ListTagsForResourceRequest;
+import software.amazon.awssdk.services.rds.model.ModifyOptionGroupRequest;
+import software.amazon.awssdk.services.rds.model.RemoveTagsFromResourceRequest;
+import software.amazon.awssdk.services.rds.model.VpcSecurityGroupMembership;
+
+public class Translator {
```

Comment:
Should we gather exception handling here or in BaseHandler class?

Emaples:
Gathered in translator: https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-codeartifact/blob/9d5e545c8e5f570ef4980f38df79ff8722360d7f/aws-codeartifact-domain/src/main/java/software/amazon/codeartifact/domain/Translator.java#L256

Gathered in Base handler: https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-apigateway/blob/5751e9513c14c1248f1b3534efb5d7358887c837/resource/src/main/java/software/amazon/apigateway/resource/BaseHandlerStd.java#L58

Reply:
Moved to BaseHandler, thank you!